src/java/org/apache/cassandra/tools/BulkLoadConnectionFactory.java

```diff
@@ -28,26 +28,24 @@
 
 public class BulkLoadConnectionFactory extends DefaultConnectionFactory implements StreamConnectionFactory
 {
-    // TODO: what is this unused variable for?
-    private final boolean outboundBindAny;
-    private final int secureStoragePort;
+    private final int storagePort;
     private final EncryptionOptions.ServerEncryptionOptions encryptionOptions;
 
-    public BulkLoadConnectionFactory(int secureStoragePort, EncryptionOptions.ServerEncryptionOptions encryptionOptions, boolean outboundBindAny)
+    public BulkLoadConnectionFactory(EncryptionOptions.ServerEncryptionOptions encryptionOptions, int storagePort)
     {
-        this.secureStoragePort = secureStoragePort;
+        this.storagePort = storagePort;
```

Comment:
we need to acutally propagate storagePort, this comes from the cli of sstableloader.